mailpoet/assets/js/src/common/thumbnail.ts

```diff
@@ -3,12 +3,12 @@ import MailPoet from 'mailpoet';
 import html2canvas from 'html2canvas';
 
 /**
- * Generates a thumbnail from a DOM element.
+ * Generates a thumbnail from a HTML element.
  *
- * @param  {DOMElement}      element
+ * @param  {HTMLElement}      element
  * @return {Promise<String>} DataURL of the generated image.
  */
-export const fromDom = async (element) => {
+export const fromDom = async (element:HTMLElement) => {
```

Comment:
What's the issue with `DomElement`, would it be a wrong type here?

Reply:
https://github.com/niklasvh/html2canvas/blob/master/src/index.ts#L19-L21
html2canvas wants explicitly an HTMLElement.

Reply:
👍